src/background/sidebar-injector.js

```diff
@@ -10,6 +10,7 @@ import { promisify } from './util';
 const CONTENT_TYPE_HTML = 'HTML';
 const CONTENT_TYPE_PDF = 'PDF';
 
+/** @param {Function} fn */
```

Comment:
Although the implementation of this helper would typecheck with most objects, the type here is documenting the intent that a function is passed, since a function's `toString()` returns its source code. That's not true for the `toString` of other objects.

As with the `injectConfig` helper, this is going to have to change completely for Manifest V3 support.

Reply:
You are right, a `Function` is a type of `object`.